src/screens/Cart/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { useContext } from "react";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+import FormInput from "components/FormImput/FormImput";
+import NoItem from "./components/NoItem";
+import PurchaseList from "./components/PurchaseList";
+import { CartContext } from "context/Cart";
+import { Info } from "interface/Purchase";
+import { INPUTS } from "./constants";
+import styles from "./styles.module.scss";
+
+function Cart() {
+  const { productList, checkout } = useContext(CartContext);
+  const { handleSubmit, control } = useForm<Info>();
+  const requiredItem = <p className={styles.requiredItem}>*</p>;
+  const onSubmit: SubmitHandler<Info> = (info) => {
+    checkout(info);
+  };
+
+  const NUMBER_VALIDATION = { min: 999999999, max: 9999999999999 };
```

Comment:
This is so wrong in so many levels but not ur fault, it's mine

Reply:
:)